modules/Users/views/view.detail.php

```diff
@@ -122,6 +122,11 @@ function preDisplay() {
                     }
                 }
             }
+            if(is_admin($current_user)){
```

Comment:
The two checks above also check for if the user is editing their own record, this shouldn't matter since if the user is locked they wont be viewing their own record but seems safer to keep this check separate?